Sources/FigmaExport/Input/Params.swift

```diff
@@ -83,6 +83,7 @@ struct Params: Decodable {
         let target: String
         let xcassetsPath: URL
         let xcassetsInMainBundle: Bool
+        let xcassetsInSwiftPackage: Bool
```

Comment:
This new parameter should be optional and false by default because:
1. Most users will not use this parameter
2. A user which updates FigmaExport to the new version will get an error because this parameter doesn't exist in the existing config file.